_pages/business-units/18f/engineering-resources/before-you-ship/security.md

```diff
@@ -0,0 +1,289 @@
+---
```

Comment:
File location probably isn't right (this document isn't specific to 18F) but it's fine for now since content will be refactored into the rest of the Handbook.

Reply:
Yep -- this is a correct suggestion for several reasons. See https://github.com/18F/handbook/issues/2532 where I document my plan once all the content lands in the Handbook successfully.